src/components/info-tile/info-tile.types.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Describes properties of circular progress bar on
+ * the info-tile
+ */
+export interface InfoTileProgress {
+    /**
+     * The value of the progress bar.
+     */
+    value: number;
+    /**
+     * The maximum value within the scale that the progress bar should visualize.
+     * Defaults to `100`.
+     */
+    maxValue?: number;
+    /**
+     * The prefix which is displayed before the `progressValue`,
+     * must be a few characters characters long.
```

Comment:
I'll fix the `characters characters` typo, but apart from that, I have the following question:

What is "must be a few characters long" supposed to mean? It's an optional property, so clearly there doesn't have to be a prefix at all. Does this mean that _if_ there is a prefix, it has to be more than one character? 🤨

Reply:
Ah! Just as I posted my question it hit me! I guess this is supposed to say something like "can be a few characters at most"?

Reply:
> "can be a few characters at most"?

yes, correct!

Reply:
I made a small change to this description.